site/public/js/rainbow-customization.js

```diff
@@ -16,6 +16,16 @@ function ExtractBuckets() {
     $('#custom_form').submit();
 }
 
+// Forces the number of expected gradeables to be greater than or equal to the current number of gradeables
```

Comment:
i think we should also add `displayerrormessage` that alerts the user why it shows this behavior.